plugins/crashes/frontend/public/javascripts/countly.views.js

```diff
@@ -140,7 +182,12 @@
                 currentTab: "stacktrace",
                 commentInput: "",
                 commentBeingEdited: undefined,
-                commentStatus: undefined
+                commentStatus: undefined,
```

Comment:
Just out of curiosity, does this initialization with "undefined" work? I mean if Vue skips it while merging it to component's main scope, it could hurt reactivity.